PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+#### What's this PR do?
+- Allows a customer to update their case
+
+#### Where should the reviewer start?
+spec/features/update_case_spec.rb
+
+
+#### Any background context you want to provide?
+We consider it sufficiently secure to allow someone with the case number to update the case
+
+#### What are the relevant tickets?
```

Comment:
I think you can probably remove this section from this repo, as you are not using Pivotal Tracker or another tracking tool.

Reply:
Or you can use Github Issues.

Basically this section is where you would link to the relevant item in your project management tool